core/trino-parser/src/test/java/io/trino/sql/parser/TestSqlParser.java

```diff
@@ -1925,6 +1927,30 @@ public void testDropCatalog()
                 new DropCatalog(location(1, 1), new Identifier(location(1, 14), "some catalog that contains space", true), false, false));
     }
 
+    @Test
+    public void testRenameCatalog()
+    {
+        assertStatement("ALTER CATALOG a RENAME TO b", new RenameCatalog(new Identifier("a"), new Identifier("b")));
+    }
+
+    @Test
+    public void testSetCatalogProperties()
+    {
+        assertStatement("ALTER CATALOG a SET PROPERTIES foo='bar'", new SetCatalogProperties(new Identifier("a"), ImmutableList.of(new Property(new Identifier("foo"), new StringLiteral("bar")))));
+        assertStatement("ALTER CATALOG a SET PROPERTIES foo=true", new SetCatalogProperties(new Identifier("a"), ImmutableList.of(new Property(new Identifier("foo"), new BooleanLiteral("true")))));
+        assertStatement("ALTER CATALOG a SET PROPERTIES foo=123", new SetCatalogProperties(new Identifier("a"), ImmutableList.of(new Property(new Identifier("foo"), new LongLiteral("123")))));
+        assertStatement("ALTER CATALOG a SET PROPERTIES foo=123, bar=456", new SetCatalogProperties(new Identifier("a"), ImmutableList.of(new Property(new Identifier("foo"), new LongLiteral("123")), new Property(new Identifier("bar"), new LongLiteral("456")))));
+        assertStatement("ALTER CATALOG a SET PROPERTIES \" s p a c e \"='bar'", new SetCatalogProperties(new Identifier("a"), ImmutableList.of(new Property(new Identifier(" s p a c e "), new StringLiteral("bar")))));
+        assertStatement("ALTER CATALOG a SET PROPERTIES foo=123, bar=DEFAULT", new SetCatalogProperties(new Identifier("a"), ImmutableList.of(new Property(new Identifier("foo"), new LongLiteral("123")), new Property(new Identifier("bar")))));
```

Comment:
@piotrrzysko do we support expressions here? Would be good to have a test for that too.